examples/flutter_gallery/test/smoke_test.dart

```diff
@@ -43,8 +43,9 @@ Future<Null> smokeDemo(WidgetTester tester, String routeName) async {
 
   await tester.tap(menuItem);
   await tester.pump(); // Launch the demo.
-  await tester
-      .pump(const Duration(seconds: 1)); // Wait until the demo has opened.
+  await tester.pump(const Duration(seconds: 1)); // Wait until the demo has opened.
+
+  expect(find.text('Flutter Gallery'), findsNothing);
```

Comment:
I feel like we've changed the case of `Flutter gallery` and Flutter Gallery at times, hopefully this is finding or not finding the right thing. :)